packages/kotti-table/src/logic/row.js

```diff
@@ -2,23 +2,22 @@ import { sortData } from './sort'
 import { updateAllSelected, cleanSelection, clearSelection } from './select'
 
 export const defaultState = {
-	_rows: [],
-	filteredRows: null,
+	_data: [],
+	filteredData: null,
+	focusedRow: null,
 	rows: null,
 }
 
 export const mutations = {
-	setRows(store, rows) {
+	setRows(store, data) {
 		const { state } = store
-		const { _rows } = state
-		const dataInstanceChanged = _rows !== rows
+		const { _data } = state
+		const dataInstanceChanged = _data !== data
 		// eslint-disable-next-line no-underscore-dangle
```

Comment:
Is this required?

Reply:
I had eslint complain when I accessed an underscored field, so yes

Reply:
Ah, I see, so it’s about `object._`, not about `_`.